inc/namespace.php

```diff
@@ -108,11 +108,22 @@ function register_column( array $columns ) : array {
 
 	$new_columns = [];
 
+	// Get the post type for this page.
+	$post_type = get_current_screen()->post_type ?? null;
```

Comment:
If we're only gathering this to pass to the filter, can we just let the callee do it instead?

Reply:
Yeah I was in 2 minds here. It's more a convenience thing as it's p likely you'll need it but yeah can do.